src/server/src/main/java/io/cassandrareaper/management/http/HttpCassandraManagementProxy.java

```diff
@@ -109,19 +111,17 @@ public String getHost() {
 
   @Override
   public List<BigInteger> getTokens() {
```

Comment:
TODO: you need to update the test for this, as it fails currently.

Reply:
Yep, I missed that one. Got it fixed now